src/parseTools.js

```diff
@@ -1040,6 +1040,14 @@ function makeHEAPView(which, start, end) {
   return 'HEAP' + which + '.subarray((' + start + ')' + mod + ',(' + end + ')' + mod + ')';
 }
 
+function makeDynCall(sig) {
+  if (!MAIN_MODULE) {
```

Comment:
In wasm dynamic linking has no JS for side modules, but for asm.js it does, so please add `&& !SIDE_MODULE`